src/lib/main.js

```diff
@@ -7,9 +7,44 @@ import I18n from '../i18n';
 import store from '../store';
 import constants from './constants';
 
+export const clearBusinessUnit = async () => {
+	const {
+		token,
+		config: existingConfig,
+	} = store.state;
+
+	Livechat.credentials.token = token;
+
+	const { departments } = await Livechat.config({
+		token,
+	});
+
+	await store.setState({ config: { ...existingConfig, departments }, businessUnit: undefined });
```

Comment:
Why are we keeping `businessUnit` prop as **undefined** rather than removing the prop?

Reply:
Actually, I went with this since we didn't have any method in the `store` to unset the property. After you pointed this out, I've added a new helper method there for the same task & used it here